service/gcsutils/remotefs/types.go

```diff
@@ -22,7 +22,7 @@ type FileInfo struct {
 	NameVar    string
 	SizeVar    int64
 	ModeVar    os.FileMode
-	ModTimeVar time.Time
+	ModTimeVar int64 // Serialization of time.Time breaks in travis, so use an int
```

Comment:
Could it be because the go version ? The Go version on the Travis is 1.8.3. What's your version?